lib/definitions/time.js

```diff
@@ -1,4 +1,5 @@
-var time
+var time;
+var daysInYear = 365.25;
```

Comment:
These aren't my changes, they're from the merge with master. [8dd137b5a3fe68a1fb6db87fd74e4827ee516068](https://github.com/ben-ng/convert-units/commit/8dd137b5a3fe68a1fb6db87fd74e4827ee516068)